extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -386,3 +386,15 @@ export async function getFileShares(account: azdata.Account, subscription: azure
 		errors: response.errors ? response.errors : []
 	};
 }
+
+export async function getMigrationControllers(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, resourceGroupName: string, regionName: string, ignoreErrors: boolean): Promise<GetMigrationControllersResult> {
+	const apiEndpoint = `https://${regionName}.management.azure.com` +
```

Comment:
I assume you're splitting up on the lines to make it easier to read - in that case it'd be better to just escape the newlines with \

```
const apiEndpoint = `https://${regionName}.management.azure.com \
    /subscriptions/${subscription.id} \
    /resourceGroups/${resourceGroupName} \
    /providers/Microsoft.DataMigration/Controllers/default/shares?api-version=2020-09-01-preview`;
```